tox.ini

```diff
@@ -18,6 +18,7 @@ whitelist_externals = make
 deps =
     django-16: Django>=1.6,<1.7
     django-17: Django>=1.7,<1.8
+    django-18: Django==1.8a1
```

Comment:
That's because 1.8a1 is technically below 1.8, `pip install --pre "Django>=1.8a1,<1.9"` works though